pkg/packagekit/updates.jsx

```diff
@@ -1587,24 +1586,22 @@ class OsUpdates extends React.Component {
             });
 
             return (
-                <>
-                    <PageSection>
-                        <Gallery className='ct-cards-grid' hasGutter>
-                            <CardsPage onValueChanged={this.onValueChanged} handleRefresh={this.handleRefresh} {...this.state} />
-                        </Gallery>
-                        { this.state.showRestartServicesDialog &&
-                            <RestartServices tracerPackages={this.state.tracerPackages}
+                <PageSection>
+                    <Gallery className='ct-cards-grid' hasGutter>
+                        <CardsPage onValueChanged={this.onValueChanged} handleRefresh={this.handleRefresh} {...this.state} />
+                    </Gallery>
+                    { this.state.showRestartServicesDialog &&
+                    <RestartServices tracerPackages={this.state.tracerPackages}
                                 close={() => this.setState({ showRestartServicesDialog: false })}
                                 state={this.state.state}
                                 callTracer={(state) => this.callTracer(state)}
                                 onValueChanged={delta => this.setState(delta)}
                                 loadUpdates={this.loadUpdates} />
```

Comment:
Same here -- either the properties should align (prefered), or it should at least have a consistent 4-space indentation).